packages/nuxt/src/vite/sourceMaps.ts

```diff
@@ -27,21 +33,25 @@ export function setupSourceMaps(moduleOptions: SentryNuxtModuleOptions, nuxt: Nu
 
   nuxt.hook('nitro:config', (nitroConfig: NitroConfig) => {
     if (sourceMapsEnabled && !nitroConfig.dev) {
-      if (nitroConfig.rollupConfig) {
-        // Add Sentry plugin
-        if (!Array.isArray(nitroConfig.rollupConfig.plugins)) {
-          nitroConfig.rollupConfig.plugins = nitroConfig.rollupConfig.plugins ? [nitroConfig.rollupConfig.plugins] : [];
-        }
+      if (!nitroConfig.rollupConfig) {
+        nitroConfig.rollupConfig = {};
+      }
+
+      if (isNullValue(nitroConfig.rollupConfig.plugins)) {
+        nitroConfig.rollupConfig.plugins = [];
+      } else if (!Array.isArray(nitroConfig.rollupConfig.plugins)) {
+        nitroConfig.rollupConfig.plugins = [nitroConfig.rollupConfig.plugins];
```

Comment:
l: Maybe we can leave a comment saying something along the lines of "`nitroConfig.rollupConfig.plugins` can be a singular plugin, in the case of which we want to put it into an array so that we can push our own plugin"